Playbooks/Linux setup.yml

```diff
@@ -1,31 +1,25 @@
 - name: Extend the logical volume to take all remaining space of the PVs and resize the underlying filesystem
   hosts: "*"
-  become: yes
+  become: true
   tasks:
-      ## Install Ansible Galaxy community collection first. "ansible-galaxy collection install community.general"
-      ## https://galaxy.ansible.com/community/general
-      community.general.lvol:
+    ## Install Ansible Galaxy community collection first. "ansible-galaxy collection install community.general"
+    ## https://galaxy.ansible.com/community/general
+    - community.general.lvol:
         vg: ubuntu-vg
         lv: ubuntu-lv
         size: +100%FREE
         resizefs: true
 
-- name: Set timezone and configure timesyncd
```

Comment:
No need to create another "play" (that's the block with name, hosts, become, tasks, roles...) for every task. In fact, it makes sense to consolidate a bit, based on what you're doing.